specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2245 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@typespec/rest";
+import "./models.tsp";
+import "./main.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Traits.Private;
+using Azure.ClientGenerator.Core;
+
+namespace PurviewDataMap;
+
+model OkResponse<T> {
+  @statusCode
+  status: 200;
+
+  @body
+  data: T;
+}
+
+op AtlasImportOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TraitContexts extends TraitContext = TraitContext.Undefined
+> is Azure.Core.Foundations.Operation<
+  TParams &
+    TraitProperties<
+      Traits &
+        NoConditionalRequests &
+        NoRepeatableRequests &
+        NoClientRequestId,
+      TraitLocation.Parameters,
+      TraitContext.Action
+    >,
+  TResponse &
+    TraitProperties<
+      Traits &
+        NoConditionalRequests &
+        NoRepeatableRequests &
+        NoClientRequestId,
+      TraitLocation.Response,
+      TraitContext.Action
+    >,
+  Traits & NoConditionalRequests & NoRepeatableRequests & NoClientRequestId,
```

Comment:
The traits in this template look redundant @timotheeguerin is this definition correct?

Reply:
@yifan-zhou922 if you update line 51 to the following do your requests still look the same:
```suggestion
  Traits,
```

Reply:
Updated Thanks